### ISSUE TITLE
can the author provide whole finetuning code?

### ISSUE THREAD

**Issue by biofocker** [GuangyuWangLab2021/Loki#9]
i see the finetune description as:
train_command = [
    'python', '-m', 'training.main',
    '--name', name,
    '--save-frequency', '5',
    '--zeroshot-frequency', '10',
    '--report-to', 'wandb',
    '--train-data', train_csv,
    '--csv-img-key', 'img_path',
    '--csv-caption-key', 'label',
    '--warmup', '10',
    '--batch-size', '64',
    '--lr', '5e-6',
    '--wd', '0.1',
    '--epochs', '10',
    '--workers', '16',
    '--model', model_name,
    '--csv-separator', ',',
    '--pretrained', pretrained_weight_path,
    '--lock-text-freeze-layer-norm',
    '--lock-image-freeze-bn-stats',
    '--coca-caption-loss-weight','0',
    '--coca-contrastive-loss-weight','1',
    '--val-frequency', '10',
    '--aug-cfg', 'color_jitter=(0.32, 0.32, 0.32, 0.08)', 'color_jitter_prob=0.5', 'gray_scale_prob=0'
]

subprocess.run(train_command)

but i can't find the "training.main" file , thus i can't do the finetune work 
I sincerely want the author provide the whole finetune code Thanks~

**Comment by osakemon** [comment]
Hi biofocker,

Thanks for your interest in our work!

Our fine-tuning process leverages the training scaffolding from the excellent [OpenCLIP](https://github.com/mlfoundations/open_clip) repository. Please head over to the [OpenCLIP GitHub](https://github.com/mlfoundations/open_clip) for the complete training scripts, configuration files, and step-by-step instructions. 

Let us know if you have other questions.

Best,
Weiqing
